src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -197,6 +197,16 @@ public function storeHasAProductPricedAt($productName, $price = 100, ChannelInte
     }
 
     /**
+     * @Given /^(this product) has been originally priced at ("[^"]+")$/
+     */
+    public function thisProductIsAlsoOriginallyPricedAtInChannel(ProductInterface $product, $originalPrice = 100, ChannelInterface $channel = null)
```

Comment:
`ChannelInterface $channel = null` is not needed.

Reply:
Also, although it's they're not used widely in **Behat** yet, we should start using strict scalar types, as we support only PHP 7.1 -> so `int $originalPrice` (I don't think the default value is needed) and `: void` at the end.